var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -65,7 +66,12 @@ class PyYt(PythonPackage):
     depends_on("py-sympy", type=('build', 'run'))
     depends_on("python @2.7:2.999,3.4:")
 
-    @run_after('install')
```

Comment:
This is not really the implementation I would have envisioned.  In the `PythonPackage` install method, I would check for (a) the existence of a variant called `develop` (or build_type; see #2692), and (b) that the variant has the appropriate value.  If both conditions hold, I would run the develop-type install procedure; otherwise the regular install.